Documentation/UserTsconfig/Options.rst

```diff
@@ -27,7 +27,7 @@ additionalPreviewLanguages
 
 :aspect:`Description`
    The user will see these additional languages when localizing stuff in
-   TCEforms. The list are uid numbers of sys_language records.
+   TCEforms. The list are ids of site languages.
```

Comment:
can we clarify this? what is an ID of a site language? is that the numeric index or something like the ISO code?

Reply:
I clarified it